### PR TITLE
Add Fortified Whiterun

### PR DESCRIPTION
**Pull request by robertgk2017**
Added Fortified Whiterun and Rob's Bug Fixes for it.

### CODE REVIEW

**Comment by robertgk2017** [comment]
Wait, why are there 2 other commits on this one?  there should ONLY be the adding fortified bits i made this on a new branch.

**Comment by sibir-ine** [comment]
You used your main branch to submit #2630, so your local commit got left over as well as a merge commit since your fork couldn't be fast-forwarded as a result. We can squash this PR into a single commit when merging, so you shouldn't need to worry. Afterwards, you can either hard reset & force push your main branch in the Git terminal or just nuke your fork & refork the repo to clean it up.

**Comment by robertgk2017** [comment]
Ah i see, i thought i hit a button on the git desktop app to add the changes to a new branch and submit.  Still figuring this out.  If that's something you can fix by all means.  I'll rebase it after this PR.

File: masterlist.yaml [hunk 1]
```diff
@@ -26970,3 +26970,22 @@ plugins:
...
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/70186/'
+        name: 'Rob''s Bug Fixes - Fortified Whiterun'
+    msg:
+      - <<: *patchUpdateAvailable
+        subs: [ '[Rob''s Bug Fixes - Fortified Whiterun](https://www.nexusmods.com/skyrimspecialedition/mods/70186/)' ]
+        condition: 'checksum("WhiterunDefenses.esp", 9A3B2ACE)'
+    dirty:
+      - <<: *quickClean
+        crc: 0x9A3B2ACE
+        util: 'SSEEdit v4.0.4c'
```
Comment (sibir-ine) [inline masterlist.yaml:26987]: This dirty cleaning data is missing a hyperlink:
```suggestion
        util: '[SSEEdit v4.0.4c](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
```

File: masterlist.yaml [hunk 2]
```diff
@@ -26970,3 +26970,22 @@ plugins:
...
+        itm: 4
+        udr: 2
+      - crc: 0xFBB83F99
```
Comment (sibir-ine) [inline masterlist.yaml:26990]: This clean cleaning data is missing a `clean` key:
```suggestion
    clean:
      - crc: 0xFBB83F99
```

File: masterlist.yaml [hunk 3]
```diff
@@ -26970,3 +26970,22 @@ plugins:
...
+        util: 'SSEEdit v4.0.4c'
```
Comment (sibir-ine) [inline masterlist.yaml:26991]: The extra newline was removed:
```suggestion
        util: 'SSEEdit v4.0.4c'

```

**Review by sibir-ine — CHANGES_REQUESTED** [review]

File: masterlist.yaml (hunk 3 above)
Comment (sibir-ine) [reply masterlist.yaml:26991]: The newline doesn't appear to have been added back in your last commit.

Comment (robertgk2017) [reply masterlist.yaml:26991]: oop, there try now, thought i entered that in.

**Review by sibir-ine — APPROVED** [review]
Thanks!

**Pull request merged**